.circleci/src/jobs/deliver_server.yml

```diff
@@ -0,0 +1,54 @@
+docker:
```

Comment:
Merged from `upload_packages_to_server` and `install_packages_to_server`

Reply:
Hm I think we had some issues that the copy was not finished properly when the Installation started.
Debian needs some time to included this in the apt Update @steffenkleinle
I think this will result to issues

Reply:
Do you think it makes sense to test this once on staging? Maybe its fixed in the meantime.

Reply:
Well there is a 60s sleep before installation.
You could adjust the install script temporary on the server for `apt install` to add `--dry-Run`
And then check the server logs if the installation dry run worked 

Or you just trigger a real new beta but i would like to avoid to then revert all packages to old release version @steffenkleinle

Reply:
Just tested, seems to work fine.